CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.13.0
+
+### Technical improvements
+
+- Integrated new rendering engine for Detection, Segmentation, Lane visualizations & AR
+- Added Okinawa as part of Japan
+
+### User experience
+
+- Added Replay & Recording modes 
+- Added reverse landscape orientation support
+- Added Lane Detection mode 
```

Comment:
To align with phrase above.
```suggestion
- Added Lane Visualization mode 
```

Reply:
Maybe `Added lane visualization to Lane Detection mode` ?

Reply:
We have then `Detection` and `Lane Detection` modes.
Not sure about it.

Reply:
`Added lane visualization to Lane Detection mode` looks ok.
Btw we really have both Object Detection and Lane Detection modes